airflow/models/__init__.py

```diff
@@ -120,6 +121,7 @@ def __getattr__(name):
     "SlaMiss": "airflow.models.slamiss",
     "TaskFail": "airflow.models.taskfail",
     "TaskInstance": "airflow.models.taskinstance",
+    "TaskInstanceHistory": "airflow.models.taskinstancehistory",
```

Comment:
Should we add this? There are already some new models we didn’t add, and since we don’t really expect people to import from this table (I think?) we should be able to afford to keep it an implementation detail.

Reply:
Good catch, originally we only had it above in `import_all_models`. Not sure when this got added.

Reply:
Removed.